Lib/test/test_string_literals.py

```diff
@@ -168,8 +167,6 @@ def test_eval_bytes_invalid_escape(self):
             eval("b'''\n\\z'''")
         self.assertEqual(len(w), 1)
         self.assertEqual(w[0].filename, '<string>')
-        if use_old_parser():
```

Comment:
Isn't this about something the peg parser doesn't do yet?

Reply:
Actually, the new parser handles this correctly. I have removed the guards in 0c61d04